### PR TITLE
Fix ifdef errors in IBZ install docs

### PR DESCRIPTION
**Pull request by mburke5678**
Fix `ifdef` errors introduced by https://github.com/openshift/openshift-docs/commit/683ffcb740739e183f96b55dee4b6e4d5c8d3734#diff-2b850a18ba10b462eb7d7822d459e49eb64b2f22db0583540d5bad25fd9e4450

### CODE REVIEW

**Comment by SNiemann15** [comment]
@mburke5678 Not sure why I can't comment directly in that part of the file (starting line 197/199 its grey and the + doesn't appear). I'm trying to explain here :-) 

The callout description for the regular installs for IBM Z should be the same as for the other arches. 
Lines 242 - 245 can be deleted
```
ifndef::restricted[]
ifdef::ibm-z,ibm-z-kvm[]
<12> The pullSecret value contains the authentication information for your registry. For `<bastion_host_name>`, specify the registry domain name that you specified in the certificate for your mirror registry, and for `<credentials>`, specify the base64-encoded user name and password for your mirror registry.
endif::ibm-z,ibm-z-kvm[]
ifndef::ibm-z,ibm-z-kvm[]
```
also line 246
`endif::ibm-z,ibm-z-kvm[]`

File: modules/installation-bare-metal-config-yaml.adoc [hunk 1]
```diff
@@ -79,7 +79,7 @@ endif::[]
 ifeval::["{context}" == "installing-ibm-power"]
   architecture : ppc64le
 endif::[]
-ifeval::["{context}" == "installing-restricted-networks-ibm-power"]
+ifeval::["{context}" == "installing-restxricted-networks-ibm-power"]
```
Comment (SNiemann15) [inline modules/installation-bare-metal-config-yaml.adoc:82]: @mburke5678 Are you sure that's correct with an "x"?

**Comment by mburke5678** [comment]
/enterprise-4.5

**Pull request merged**